google/cloud/bigquery/table.py

```diff
@@ -1836,6 +1837,17 @@ def to_arrow_iterable(
                 created by the server. If ``max_queue_size`` is :data:`None`, the queue
                 size is infinite.
 
+            max_stream_count (Optional[int]):
```

Comment:
I think it would be more consistent if we use the same docstring as [here](https://github.com/googleapis/python-bigquery/blob/a4d9534a900f13ae7355904cda05097d781f27e3/google/cloud/bigquery/_pandas_helpers.py#L1083-L1087). It also mentions the effect of `preserve_order` (in this case `self._preserve_order`), which I think we should make clear here.

Reply:
Updated